src/api/java/com/ldtteam/perviaminvenire/api/pathfinding/AbstractPathJob.java

```diff
@@ -140,24 +141,29 @@ public AbstractPathJob(
      * @param entity           the entity.
      * @see AbstractPathJob#AbstractPathJob(World, BlockPos, BlockPos, int, LivingEntity)
      */
-    public AbstractPathJob(final World world, final BlockPos startRestriction, final BlockPos endRestriction, final PathResult<? extends AbstractPathJob> result, final LivingEntity entity)
+    public AbstractPathJob(final World world,
+        final BlockPos start,
+        final BlockPos startRestriction,
+        final BlockPos endRestriction,
+        final int range,
+        final Vector3i grow,
+        final PathResult<AbstractPathJob> result,
+        final LivingEntity entity)
```

Comment:
Document the new parameters in the JavaDoc of the constructor.
Pay particular attention to `grow` since i find it a that bit confusing.

Reply:
I've forgot entire jdoc lol